concrete/testtool/testtool.go

```diff
@@ -43,84 +42,81 @@ var (
 	PrintLogs = true
 )
 
-func runTestMethod(concreteRegistry concrete.PrecompileRegistry, bytecode []byte, method abi.Method, shouldFail bool) (uint64, error) {
-	var (
-		key, _          = crypto.HexToECDSA("d17bd946feb884d463d58fb702b94dd0457ca349338da1d732a57856cf777ccd") // 0xCcca11AbAC28D9b6FceD3a9CA73C434f6b33B215
-		senderAddress   = crypto.PubkeyToAddress(key.PublicKey)
-		contractAddress = common.HexToAddress("cc73570000000000000000000000000000000000")
-		gspec           = &core.Genesis{
+func runTestMethod(t *testing.T, concreteRegistry concrete.PrecompileRegistry, bytecode []byte, method abi.Method, shouldFail bool) {
+	t.Run("runTestMethod", func(t *testing.T) {
```

Comment:
What is this sub-test for? And what's the reasoning behind taking the variable declarations out of the 'var ()` clause?